patches/v8/BUILD.gn.patch

```diff
@@ -1,7 +1,16 @@
 diff --git a/BUILD.gn b/BUILD.gn
-index ea4eddb93336b511d1007ab9f3853c5429f2dbff..036ed50b7ae542c2b8234f7e6b43f6dbe6a97f6f 100644
+index ea4eddb93336b511d1007ab9f3853c5429f2dbff..1c662b5044dbc808d55e39b34bc1388452cc94e6 100644
 --- a/BUILD.gn
 +++ b/BUILD.gn
+@@ -407,7 +407,7 @@ declare_args() {
+   v8_scriptormodule_legacy_lifetime = false
+ 
+   # WebAssembly interpreter (DrumBrake)  build flag.
+-  v8_enable_drumbrake = false
++  v8_enable_drumbrake = true
```

Comment:
this does not require a patch because it's in declare_args. Just add it to config.js

Reply:
Resolved locally